libs/blocks/global-navigation/global-navigation.js

```diff
@@ -512,9 +512,10 @@ class Gnav {
       return 'linux';
     };
 
+    const unavVersion = new URLSearchParams(window.location.search).get('unav');
```

Comment:
This should be easier
```suggestion
    const unavVersion = new URLSearchParams(window.location.search).get('unav') || '1.1';
```